modules/adheseBidAdapter.js

```diff
@@ -18,13 +18,16 @@ export const spec = {
     if (validBidRequests.length === 0) {
       return null;
     }
+    const gdprConsent = bidderRequest.gdprConsent;
```

Comment:
You could use destructuring to combine lines 21 - 22 into this:
`const { gdprConsent, refererInfo } = bidderRequest;`